pyproject.toml

```diff
@@ -0,0 +1,44 @@
+[tool.poetry]
+name = "npc-gzip"
+version = "0.1.0"
+description = "Code for Paper: “Low-Resource” Text Classification: A Parameter-Free Classification Method with Compressors"
+authors = [
+    "Gin Jiang <bazingagin@gmail.com>",
+    "Zach Bloss <zacharybloss@gmail.com>",
+]
+maintainers = [
+    "Zach Bloss <zacharybloss@gmail.com>",
+    "Gin Jiang <bazingagin@gmail.com>",
+]
+license = "MIT"
+readme = "README.md"
+homepage = "https://github.com/bazingagin/npc_gzip"
+repository = "https://github.com/bazingagin/npc_gzip"
+keywords = ["npc_gzip", "knn-gzip"]
+packages = [{include = "npc_gzip"}]
+
+[tool.poetry.dependencies]
+python = "^3.9"
+numpy = "^1.25.1"
```

Comment:
A bugfix update, numpy [1.25.2](https://github.com/numpy/numpy/releases/tag/v1.25.2), is now available.